### PR TITLE
Light driver for PCA9632

### PR DESCRIPTION
**Pull request by kaedwen**
## Description:

This PR adds a raw PCA9632 driver and a light driver for that device. It can be configured with the `USE_PCA9632` defines. The I2C address and the channel mapping can be configured.

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [ ] The code change is tested and works with Tasmota core ESP32 V.2.0.6
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

File: tasmota/my_user_config.h [hunk 1]
```diff
@@ -632,6 +632,12 @@
 //  #define USE_PCA9685                            // [I2cDriver1] Enable PCA9685 I2C HW PWM Driver - Must define I2C Address in #define USE_PCA9685_ADDR below - range 0x40 - 0x47 (+1k4 code)
 //    #define USE_PCA9685_ADDR 0x40                // Enable PCA9685 I2C Address to use (Must be within range 0x40 through 0x47 - set according to your wired setup)
 //    #define USE_PCA9685_FREQ 50                  // Define default PWM frequency in Hz to be used (must be within 24 to 1526) - If other value is used, it will rever to 50Hz
+//  #define USE_PCA9632                            // [I2cDriver91] Enable PCA9632 I2C HW PWM Driver
```
Comment (arendst) [inline tasmota/my_user_config.h:635]: Should be I2cDriver75

File: tasmota/tasmota_xdrv_driver/xdrv_91_pca9632.ino [hunk 2]
```diff
@@ -0,0 +1,275 @@
+/*
+  xdrv_15_pca9632.ino - Support for I2C PCA9632 4bit 8 pin hardware PWM driver on Tasmota
```
Comment (arendst) [inline tasmota/tasmota_xdrv_driver/xdrv_91_pca9632.ino:2]: Should be xdrv_91_pca9632.ino

File: tasmota/my_user_config.h (hunk 1 above)
Comment (kaedwen) [reply tasmota/my_user_config.h]: fixed to `I2cDriver75`

File: tasmota/tasmota_xdrv_driver/xdrv_91_pca9632.ino (hunk 2 above)
Comment (kaedwen) [reply tasmota/tasmota_xdrv_driver/xdrv_91_pca9632.ino]: fixed to `xdrv_91_pca9632.ino`

**Pull request merged**